Library/Formula/libtorrent-rasterbar.rb

```diff
@@ -13,10 +13,6 @@ class LibtorrentRasterbar < Formula
   end
 
   bottle do
-    cellar :any
```

Comment:
No worries :). Essentially the test-bot we have in place creates new bottles for the formula, and then when a maintainer pulls the update with ` brew pull --bottle ` the bottles are published on Bintray and existing bottle block is replaced with the new block.